_includes/samples/sdl2_ttf/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ target_include_directories(${PROJECT_NAME} PRIVATE
 target_link_libraries(${PROJECT_NAME} PRIVATE
     ${SDL2_LIBRARIES}
     ${SDL2_TTF_LIBRARIES}
+    stdc++
```

Comment:
Why do we need stdc++? Because of harfbuzz? Then it should be fixed in the pkgconf for harfbuzz.

Reply:
Yes the harfbuzz, agreed that should be fixed. Got it let me close this PR

Reply:
Cool, thanks for making this PR, though. It's appreciated nonetheless.